tracer/test/Datadog.Trace.Tests/Configuration/DynamicConfigurationTests.cs

```diff
@@ -42,27 +42,69 @@ public void ApplyConfigurationTwice()
         {
             var tracer = TracerManager.Instance;
 
-            DynamicConfigurationManager.OnlyForTests_ApplyConfiguration(CreateConfig((ConfigurationKeys.GlobalSamplingRate, "0.4")));
+            DynamicConfigurationManager.OnlyForTests_ApplyConfiguration(CreateConfig(("tracing_sampling_rate", "0.4")));
 
             var newTracer = TracerManager.Instance;
 
             newTracer.Should().NotBeSameAs(tracer);
 
-            DynamicConfigurationManager.OnlyForTests_ApplyConfiguration(CreateConfig((ConfigurationKeys.GlobalSamplingRate, "0.4")));
+            DynamicConfigurationManager.OnlyForTests_ApplyConfiguration(CreateConfig(("tracing_sampling_rate", "0.4")));
 
             TracerManager.Instance.Should().BeSameAs(newTracer);
         }
 
+        [Fact]
+        public void ApplyTagsToDirectLogs()
+        {
+            var tracerSettings = new TracerSettings();
```

Comment:
It might be nice to check that the dynamic values are recorded in the telemetry. I don't think we have any tests for that currently 🤔 

You can see some examples of how you could test that in `ConfigurationTelemetryCollectorTests`

Reply:
We do (`AssertConfigurationChanged` in DynamicConfigurationTests)